packages/components/src/components/hds/pagination/numbered/index.ts

```diff
@@ -59,19 +109,18 @@ export const elliptize = ({ pages, current, limit = 7 }) => {
     const delta = (limit - 5) / 2; // this is why the limit needs to be an odd number
     // we slice the array starting at the "current" index, minus the delta, minus one because it's an array (zero-based)
     const sliceCurr = pages.slice(current - delta - 1, current + delta);
-    result = [].concat(
-      sliceStart.shift(),
-      ellipsis,
+    result = ([] as HdsPaginationElliptizedPageArray).concat(
```

Comment:
Probably out of scope for this PR, but its odd to me that we support both number and string for these... Should we consider scoping it down to just number? Is there a specific reason we allow both?

Reply:
Ahh... ok I misread and thought it was either a number or a string of a number.. maybe we can limit the type to be `number | ELLIPSIS` ? that way the ellipsis definition can be moved to the types file which is more inline with how we define re-usable strings?

Reply:
I did try that, but it causes some issues if we try to initialize an empty object that uses keys that are of that type.

For instance:

```
queryPages: Record<HdsPaginationElliptizedPageArrayItem, string> = {};
```

The above will work if the type of `HdsPaginationElliptizedPageArrayItem` is `string | number`, but not if its `typeof ELLIPSES | number`